app/src/main/java/org/metabrainz/android/presentation/features/userprofile/collections_section/CollectionsSectionScreen.kt

```diff
@@ -0,0 +1,9 @@
+package org.metabrainz.android.presentation.features.userprofile.collections_section
+
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+
+@Composable
+fun CollectionsSectionScreen(){
```

Comment:
Since this work isn't final yet, I suggest we base it on another branch and not master so that we avoid merging incomplete work to master for now. All looks great btw!